File: code-coaches/easy/skee-ball/skee_ball.py
def can_buy_squirt_gun(score: int, squirt_gun_price: int) -> bool:
    ratio = 12  # 12 points = 1 ticket
    tickets = score / ratio
    return tickets >= squirt_gun_price


def main():
    score = int(input())
    squirt_gun_price = int(input())

    if can_buy_squirt_gun(score, squirt_gun_price):
        print("Buy it!")
    else:
        print("Try again")


if __name__ == "__main__":
    main()

File: code-coaches/easy/skee-ball/SkeeBall.java
import java.util.Scanner;

public class SkeeBall {

    public static boolean canBuySquirtGun(int score, int squirtGunPrice) {
        int ratio = 12; // 12 points = 1 ticket
		int tickets = score / ratio;
		return tickets >= squirtGunPrice;
    }

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int score = input.nextInt();
		int squirtGunPrice = input.nextInt();

        if (canBuySquirtGun(score, squirtGunPrice)) {
            System.out.print("Buy it!");
        } else {
            System.out.print("Try again");
        }

		input.close();
	}

}
